### ISSUE TITLE
Need help : AllowClipboard doesn't work  the --force-renderer-accessibility chrome option

### ISSUE THREAD

**Issue by fhachelaf** [itsmikeeng/AllowClipboard#1]
I would like to use your extension with Chrome 45 in a professional context.
After testing it  I would like to ask you 2 things :

1/ Is it possible to have a version of the AllowClipboard extension without the following confirmation message : 'Allow this webpage to access your Clipboard?'

2/ When I run Chrome.exe with the --force-renderer-accessibility option, the copy operation doesn't   work every time and not at all with a large volume of data.
 Do you have an idea of the reason of this problem ?

Thanks
Cordially

**Comment by itsmikeeng** [comment]
Hi fhachelaf,
1. The "Allow this webpage to access your Clipboard" prompt is needed so that a user's clipboard isn't accessible by any webpage without their permission. I could potentially add a Chrome Extension Option (https://developer.chrome.com/extensions/options) to optionally disable it, but it seems like a security risk to disable it...
2. I wasn't able to reproduce, could you provide repro steps?
   I tried by launching `C:\Program Files (x86)\Google\Chrome\Application\chrome.exe --force-renderer-accessibility`. Then used the `AllowClipboard.Client.ClipboardClient()` from the Chrome Developer Console (ctrl-shift-I). I was able to use the example from the main page of this GitHub project, as well as copy large amount of text fine (+1000000 characters).

Thanks,
itsmikeeng
